### PR TITLE
feat: export functions for converting between Date object and string

### PR DESCRIPTION
**Pull request by insberr**
## Checklist

Please put "X" in the below checkboxes that apply:

- [ ] If committing a change of production code, I have tested it in different browsers (Chrome, Firefox, Safari). 
- [ ] If committing a new feature, I have first submitted an issue (Please note: you are free to open PRs for non-issued features, but opening an issue increases your chance of a successful PR). 
- [ ] If committing a new feature, I have also written the appropriate tests for it. 
- [ ] I have tried to build the feature in a way, that it does not cause any breaking changes for others (unless 
  agreed upon in an issue). 

**Premium**
- [ ] I'm a Schedule-X premium user

## This PR solves the following problem**. 
Export `toDateString`, `toTimeString`, `toDateTimeString`, and `toJSDate` so users can use `Date` objects or parse the date string into Date objects.

## How to test this PR**. 
1. Make sure `toDateString`, `toTimeString`, and `toJSDate` are available when importing.
2. Call the functions and make sure they work (I assume they do work)

### CODE REVIEW

File: packages/calendar/src/index.ts [hunk 1]
```diff
@@ -7,7 +7,12 @@ import {
   CalendarConfigExternal as CalendarConfig,
   CustomComponentFn,
 } from '@schedule-x/shared/src/interfaces/calendar/calendar-config'
-import { CalendarEvent } from '@schedule-x/shared/src'
+import {
+  CalendarEvent,
```
Comment (tomosterlund) [inline packages/calendar/src/index.ts:11]: I think you will probably need `toDateTimeString` too :)

Comment (tomosterlund) [reply packages/calendar/src/index.ts:11]: The one converts into full-day events with the format YYYY-MM-DD, and the other into timed events with the format YYYY-MM-DD HH:mm

Comment (insberr) [reply packages/calendar/src/index.ts:11]: Oh it was hidden haha, ill add that real quick.

**Comment by insberr** [comment]
Oh no, some checks failed. I can't tell if my changes caused them to fail though, it looks like there was some other issue?

**Comment by tomosterlund** [comment]
> Oh no, some checks failed. I can't tell if my changes caused them to fail though, it looks like there was some other issue?

I think this is unrelated to, but potentially triggered by, these changes. I can have a look in a moment :)

**Pull request merged**